SecondCours/HW1/task6.py

```diff
@@ -0,0 +1,69 @@
+# Задание 6. Уровень - Кодзима, ты ли это?
+# Условие: вы создали игру в жанре шутер. Теперь ваш дизайнер придумал новое неизвестное никому оружее - дробовик.
+# Известно, что дробовики стреляют дробью (внезапно, правда?). Ваша задача - рассчитать суммарный урон, наненсенный
+# выстрелом из дробовика.
+
+
+# Входные данные : Сначала вводится количество дробинок.
+# Затем урон от каждой дробинки. Урон от каждой дробинки выражается простой дробью,
+# её числитель и знаменатель вводятся на отдельных строках.
+
+# Выходные данные : Суммарный урон, выраженный простой несократимой дробью с дробной
+# чертой между числителем и знаменателем.
+
+
+# Пример:
+# Ввод:                                               # Вывод:
+# 2
+# 1
+# 50
+# 3
+# 20                                                 17/100
+
+
+# 3
+# 1
+# 50
+# 2  1
+# 40 20
+# 3  1
+# 30 10                                                17/100
+
+
+from collections import namedtuple
```

Comment:
task6 --- несоответствие стандарту PEP8,
```
ek.py:26:0: C0301: Line too long (110/100) (line-too-long)
ek.py:32:0: C0301: Line too long (105/100) (line-too-long)
ek.py:35:0: C0304: Final newline missing (missing-final-newline)
ek.py:1:0: C0111: Missing module docstring (missing-docstring)
ek.py:11:8: W0621: Redefining name 'i' from outer scope (line 31) (redefined-outer-name)
ek.py:7:0: C0103: Argument name "a" doesn't conform to snake_case naming style (invalid-name)
ek.py:7:0: C0103: Argument name "b" doesn't conform to snake_case naming style (invalid-name)
ek.py:7:0: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
ek.py:22:8: W0621: Redefining name 'i' from outer scope (line 31) (redefined-outer-name)
ek.py:22:4: C0200: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)
ek.py:24:4: C0200: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)
ek.py:30:4: C0103: Constant name "total" doesn't conform to UPPER_CASE naming style (invalid-name)
ek.py:34:4: C0103: Constant name "dmg" doesn't conform to UPPER_CASE naming style (invalid-name)
```